scripts/generate-app-config/generateContentConfig.js

```diff
@@ -0,0 +1,225 @@
+const kebabCase = require('lodash.kebabcase');
+const startCase = require('lodash.startcase');
+const ImportAggregator = require('./generation-objects/ImportAggregator');
+
+// "require" items to be added to the generated config.
+const ContentWrapper = 'terra-dev-site/lib/wrappers/_ContentWrapper';
+const MarkdownWrapper = 'terra-dev-site/lib/wrappers/_MarkdownWrapper';
+const TerraScreenshotWrapper = 'terra-dev-site/lib/wrappers/_ScreenshotWrapper';
+
+const routeImporter = new ImportAggregator();
+/**
+* Setup a menuItem object.
+*/
+const menuItem = (text, itemPath, hasSubMenu, childItems) => ({
+  text,
+  path: itemPath,
+  childItems,
+  ...(hasSubMenu) && { hasSubMenu: true },
+});
+
+/**
+ * Builds out a route item. Adds the props object conditionally.
+ */
+const routeItem = (text, routePath, { contentPath, name, identifier }, props) => ({
+  text,
+  path: routePath,
+  component: {
+    default: {
+      componentClass: routeImporter.addImport(ImportAggregator.relativePath(contentPath), name, identifier),
+      ...(props) && { props },
+    },
+  },
+});
+
+const evidenceProps = (contentConfig) => {
+  const contentCopy = Object.assign({}, contentConfig);
+  return {
+    imageConfig: Object.keys(contentCopy).reduce((acc, viewportKey) => {
+      const viewport = contentCopy[viewportKey];
+      const contentPath = routeImporter.addImport(ImportAggregator.relativePath(viewport));
+      acc.push({
+        viewport: viewportKey,
+        contentPath,
+      });
+      return acc;
+    }, []),
+  };
+};
+
+/**
+ * Sets up content route item. All content items are wrapped with the content wrapper.
+ */
+const contentRouteItem = (text, routePath, { contentPath, name, identifier }, props, type) => {
+  let contentProps;
+  let content = { contentPath: ContentWrapper, name: 'ContentWrapper' };
+  if (typeof contentPath === 'string') {
```

Comment:
What other value could contentPath be?

Reply:
I think content path could be an actual inline react component.

Reply:
maybe? I'm going to be honest, i can't remember why we're checking for a string here but i'm hesitant to change it.

Reply:
I'll remove this.